### PR TITLE
Move samples into single application

### PR DESCRIPTION
**Pull request by Selindek**

### CODE REVIEW

File: projects/samples/src/app/post-statement/post-statement.component.ts [hunk 1]
```diff
@@ -34,7 +31,7 @@ export class AppComponent {
 
     this.client.postStatement(statement).subscribe({
       next: (response) =>
-        (this.response = response.body ? response.body[0] : undefined),
+        (this.response = response.body ? response.body : undefined),
```
Comment (thomasturrell) [inline projects/samples/src/app/post-statement/post-statement.component.ts:34]: unnecessary

File: projects/samples/src/app/get-state/get-state.component.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    this.client
+      .getState<object>({
+        activityId: 'https://example.com/activity',
+        agent: {
+          mbox: 'mailto:another@example.com',
+        },
+        stateId: 'progress',
+      })
+      .subscribe({
+        next: (response) => (this.response = JSON.stringify(response.body)),
```
[the remark this answers is not in the document]
Comment (Selindek) [reply projects/samples/src/app/get-state/get-state.component.ts:27]: Yes, but the stored object (in postState) is an object, so it's nicer if we get it as an object. I transform it to a string only becasue that's the simplest way to render it in html.

[the remark this answers is not in the document]
Comment (Selindek) [reply projects/samples/src/app/get-state/get-state.component.ts:27]: ... and the sample can demonstrate that it can get an object directly this way.

**Review by thomasturrell — APPROVED** [review]

File: projects/samples/src/app/app.module.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    PostStateComponent,
+    GetStateComponent,
+  ],
+  imports: [BrowserModule, HttpClientModule],
+  providers: [
+    {
+      provide: XAPI_CONFIG,
+      useValue: {
+        endpoint: 'https://lrs.adlnet.gov/xapi/',
+        authorization: 'Basic eGFwaS10b29sczp4YXBpLXRvb2xz',
```
Comment (thomasturrell) [inline projects/samples/src/app/app.module.ts:26]:
```suggestion
        authorization: '',
```

**Review by thomasturrell — APPROVED** [review]

**Pull request merged**